app/components/layout/sidebar/bottom.tsx

```diff
@@ -1,43 +1,114 @@
 }
 
 export default function SidebarBottom({ user }: Props) {
-  const [, toggleMobileNav] = useAtom(toggleMobileNavAtom);
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+
   return (
-    <div className="bottom gap-2">
-      <Link
-        to="settings"
-        className="flex-1 rounded-lg p-1 hover:bg-gray-100"
-        onClick={toggleMobileNav}
-      >
-        <div className="flex items-center gap-3">
-          <ProfilePicture width="w-10" height="h-10" />
-          <div className="user-credentials flex-1 text-[14px] transition-all duration-200 ease-linear">
-            <div className="line-clamp-1 block text-ellipsis font-semibold">
-              {user.username}
+    <div className="bottom w-full gap-2">
+      <DropdownMenu modal={false}>
+        <DropdownMenuTrigger
+          onClick={() => setDropdownOpen((prev) => !prev)}
+          className="outline-none focus-visible:border-0"
+        >
+          <div className="flex w-full items-center justify-between gap-x-5 rounded-lg border-[1px] border-gray-200 p-3 hover:bg-gray-100">
+            <div className="flex gap-3">
+              <ProfilePicture width="w-10" height="h-10" />
+              <div className="user-credentials flex-1 text-left text-[14px] transition-all duration-200 ease-linear">
+                <div className="line-clamp-1 block text-ellipsis font-semibold">
+                  {user.username}
+                </div>
+                <p
+                  className="line-clamp-1 block text-ellipsis"
+                  data-test-id="userEmail"
+                >
+                  {user.email}
+                </p>
+              </div>
             </div>
-            <p
-              className="line-clamp-1 block text-ellipsis"
-              data-test-id="userEmail"
+
+            <i
+              className={`inline-block px-3 py-0 text-gray-400 transition-all duration-300 ease-out ${
+                dropdownOpen && "rotate-180"
+              }`}
             >
-              {user.email}
-            </p>
+              <ChevronDown />
+            </i>
           </div>
-        </div>
-      </Link>
+        </DropdownMenuTrigger>
 
-      <LogoutButton
-        className="logout-btn h-8 w-8 transition-all duration-200 ease-linear"
-        onClick={toggleMobileNav}
-      />
+        <DropdownMenuContent
+          align="end"
+          className="order w-[280px] rounded-md bg-white p-0 text-right "
+        >
+          <DropdownMenuItem className="border-b-[1px] border-gray-200 px-4 py-3">
+            <Button
+              to={`settings/account`}
+              icon="profile"
+              role="link"
+              variant="link"
+              className="justify-start text-gray-700 hover:text-gray-700"
+              width="full"
+            >
+              Account Details
+            </Button>
+          </DropdownMenuItem>
+          <DropdownMenuItem className="px-4 py-3">
+            <Button
+              icon="question"
```

Comment:
@rajdip-b 

- Remove 'Leave feedback' button
- Change 'Contact support' in to 'Questions/Feedback'
- 'Questions/Feedback' should open Crisp chat like it does now

Reply:
not sure if im getting this. points 1 and 3 are clear. changing contact support to questions/feedback means removing a button entirely. it would be rather nice if you can give me a list of items that will be present in the dropdown! @jurrejansen

Reply:
@rajdip-b Sure!

Dropdown:
- Account details
- Questions/Feedback
- Log out

![Screenshot 2023-12-18 at 13 35 19](https://github.com/Shelf-nu/shelf.nu/assets/76112272/1504b779-8369-470b-ac0e-55814d35dd1e)